test/espnet2/enh/decoder/test_stft_decoder.py

```diff
@@ -29,6 +37,7 @@ def test_STFTDecoder_backward(
         center=center,
         normalized=normalized,
         onesided=onesided,
+        spec_transform_type=spec_transform_type,
```

Comment:
Isn't it more flexible to specify a lambda function here ?

Reply:
Mmmh it would be difficult however to specify from the YAML it needs to be a function on complex tensor. 
IDK then maybe leave it as it is

Reply:
I see: ). Let's just leave it as it is.